cloudinstall/machine.py

```diff
@@ -126,8 +126,10 @@ def containers(self):
         :rtype: generator
         """
         _containers = self.machine.get('Containers', {}).items()
+        result = []
         for container_id, container in _containers:
-            yield Machine(container_id, container)
```

Comment:
the return from the api call has apparently changed and yield a Machine class doesn't work the same for containers. we can just use the dict or eventually create an additional container class.